packages/inventory/src/EntityTableToolbar.js

```diff
@@ -111,77 +105,47 @@ class ContextEntityTableToolbar extends Component {
-                        this.debounceGetAllTags(value, { filters });
-                    });
-                },
-                onChange: (_e, newSelection, group, item, groupKey, itemKey) => {
-                    if (item.meta) {
-                        const isSelected = newSelection[groupKey][itemKey];
-                        newSelection[groupKey][itemKey] = {
-                            isSelected,
-                            group,
-                            item
-                        };
-                        this.setState(
-                            { selected: newSelection },
-                            () => {
-                                const newFilter = this.applyTags(newSelection);
-                                getAllTags(filterTagsBy, { filters: newFilter });
-                            });
-                    }
-                },
-                selected,
-                ...allTagsLoaded && allTags.length > 0 ? {
-                    groups: [
-                        ...constructGroups(allTags),
-                        ...additionalTagsCount > 0 ? [{
-                            items: [{
-                                label: `${additionalTagsCount} more tags available`,
-                                isDisabled: true,
-                                className: 'ins-c-inventory__tags-more-items'
-                            }]
-                        }] : []
-                    ]
-                } : {
-                    items: [
-                        {
-                            label: !allTagsLoaded ? <Fragment>
-                                <span>
-                                    Loading... <Spinner size="md" />
-                                </span>
-                            </Fragment> : <div className="ins-c-inventory__tags-no-tags">
-                                No tags available
-                            </div>,
-                            isDisabled: true,
-                            className: 'ins-c-inventory__tags-tail'
-                        }
-                    ]
-                }
-            }
-        };
+        const { allTags, allTagsLoaded, additionalTagsCount, filters, toggleTagModal } = this.props;
+        const { selected } = this.state;
+        return tagsFilterBuilder(
+            (value) => this.setState(
+                { filterTagsBy: value },
+                () => this.debounceGetAllTags(value, { filters })
+            ),
+            this.updateSelectedTags,
+            selected,
+            allTagsLoaded,
+            allTags,
+            additionalTagsCount > 0 ? [{
+                items: [{
+                    label: `${additionalTagsCount} more tags available`,
+                    onClick: () => toggleTagModal(),
+                    className: 'ins-c-inventory__tags-more-items'
+                }]
+            }] : [],
+            <span> Loading... <Spinner size="md" /> </span>
```

Comment:
can we get away with only a spinner here? (sans text)

Reply:
We should be able to remove the text here, the Spinner has hidden text for screen readers